app/code/Magento/ImportExport/Block/Adminhtml/Export/Filter.php

```diff
@@ -418,14 +437,45 @@ public function getRowUrl($row)
         return false;
     }
 
+    /**
+     * @return array
+     * @throws \Magento\Framework\Exception\LocalizedException
+     */
+    public function getDisabledAttrs()
```

Comment:
please change the method access level to private and preferable rename function to more verbose `getDisabledAttributes`

Reply:
sure :)